modules/widget-visibility/widget-conditions/widget-conditions.js

```diff
@@ -200,32 +200,36 @@ jQuery( function( $ ) {
 			select = condition.find( '.conditions-rule-minor' ).html( '' ),
 			major = condition.data( 'rule-major' );
 
-		if ( ! major ) {
+		if ( ! major || major === 'post_type' ) {
```

Comment:
Can you please modify the condition to be Yoda-style `'post_type' ===`. Also, let's add a comment before this condition that it's done for backwards compatibility in order to avoid errors when people have "Post Type" selected?